examples/QIR/QIR.sln

```diff
@@ -5,6 +5,38 @@ VisualStudioVersion = 16.0.29613.14
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Emission", "Emission\Emission.csproj", "{A8D9184C-C211-42D2-9F48-4D2DBE810A1C}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Development", "Development\Development.csproj", "{94ECCC2D-96E7-4CFE-A37F-37266C091BC1}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Compiler", "Compiler", "{D1D740B2-1AC8-46EC-9192-4C8117AAE345}"
```

Comment:
I am curious about why many of these projects have to be part of this solution if they are already part of the compiler solution?

Reply:
I would love to not have them, but the development project has a reference to the compiler and the QIR generation such that it can use the source code version to compile the project. Unfortunately, I have to have all the project dependencies in the same solution for the build in VS to work correctly; otherwise the build can fail with a somewhat cryptic error.

Reply:
It looks like what would be possible to do is to check in the .suo file in the hidden .vs folder to hide the solution folders (all compiler projects) from the solution view entirely. I am not sure if this is a good thing to do; if we do that and someone wants to add one of these projects to the solution (e.g. because they want to edit and test the QIR generation), then they get a somewhat confusing error of "the project already exists in the solution" despite that the solution doesn't look like it contains that project. It also seems that the solution folders (all compiler projects are in a solution folder, same for the qir generation projects) are closed by default such that the first view that people see is a solution with two folders and two project.